Dockerfile.multi

```
@@ -1,25 +1,36 @@
 # Dockerfile.multi
-# v0.7.5
+# v0.7.7-rc1
 
 # Base for all builds
-FROM node:20-alpine AS base
+FROM node:20-alpine AS base-min
 WORKDIR /app
 RUN apk --no-cache add curl
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)